### PR TITLE
Refactor EMLaide for improved API 

### PR DESCRIPTION
**Pull request by ErinCain**
Refactors documentations and functions to create new api using add_[blank] wrappers for create_[blank] functions.

### CODE REVIEW

File: data-raw/make_metadata_eml.R [hunk 1]
```diff
@@ -0,0 +1,60 @@
+library(EMLaide)
```
Comment (thesadie) [inline data-raw/make_metadata_eml.R]: should this file be in this repo?

File: R/raster-element.R [hunk 2]
```diff
@@ -76,20 +75,71 @@ add_raster <- function(parent_element, file_name, file_description, attribute_li
...
 }
+#' Add Raster File
+#' @description Adds the raster elements to a dataset list according to EML standards. 
+#' @param parent_element A list representing the EML project or dataset.
+#' @param raster_metadata A named list or dataframe containing raster metadata elements
+#' (file_name, file_description, attribute_list, physical, spatial_reference, horizontal_accuracy, vertical_accuracy, 
+#' cell_size_x, cell_size_y, number_of_bands, raster_origin, rows, columns, verticals, cell_geometry): see \code{\link{create_raster}} 
+#' @return The dataset list or project with raster file information appended.
+#' @examples  
+#' raster_metadata <- list("file_name" = "Rasterfiles.zip" ,
```
Comment (thesadie) [inline R/raster-element.R]: why .zip for the file?

File: R/taxonomic-coverage-element.R [hunk 3]
```diff
@@ -40,17 +40,17 @@
 #' name taxonomic_coverage to append it to the dataset or project. Example of how
```
Comment (thesadie) [inline R/taxonomic-coverage-element.R:40]: lines 29-33 contain the ITIS identifier, pls add to EMLaid::CVPIA_common_species so the warning goes away

File: _pkgdown.yml [hunk 4]
```diff
@@ -14,26 +14,42 @@ home:
     href: http://cvpia.scienceintegrationteam.com/
     
 reference:
-- title: EML Element Functions
+- title: Create EML Element Functions
+  desc: Functions for creating metadata sections.
+  contents:
+  - create_abstract
```
Comment (thesadie) [inline _pkgdown.yml:20]: plz, confirm that all functions that need to be here are

**Review by thesadie — CHANGES_REQUESTED** [review]
I think that some of the `#' @examples` should be wrapped in `\dontrun`, let's discuss tomorrow. I made slight changes to the docs here and there. I did not rebuild the docs or pkgdown site. Plz do after we finalize examples.

**Comment by thesadie** [comment]
@ErinCain tests seem broken, are it could be my machine. let's check in on monday about this

File: data-raw/make_metadata_eml.R (hunk 1 above)
Comment (ErinCain) [reply data-raw/make_metadata_eml.R]: I deleted. I was using it to test functions when refactoring but it does not need to be here anymore.

File: R/taxonomic-coverage-element.R (hunk 3 above)
Comment (ErinCain) [reply R/taxonomic-coverage-element.R:40]: Fixed warning

File: _pkgdown.yml (hunk 4 above)
Comment (ErinCain) [reply _pkgdown.yml:20]: Confirmed.

File: R/raster-element.R (hunk 2 above)
Comment (ErinCain) [reply R/raster-element.R]: Changed to .tif

**Pull request merged**